src/main.rs

```diff
@@ -93,55 +93,57 @@ fn print_file<P: AsRef<Path>>(
     line_changes: &Option<LineChanges>,
 ) -> Result<()> {
     let mut highlighter = HighlightFile::new(filename.as_ref(), syntax_set, theme)?;
+    let mut pager = Command::new("less").arg("-FRX").stdin(Stdio::piped()).spawn()?;
```

Comment:
If there are long option alternatives for these flags (like `-F` => `--quit-if-one-screen`), I would prefer to use those for readability purposes.